crates/ruff_python_semantic/src/cfg/graph.rs

```diff
@@ -223,10 +227,30 @@ impl<'stmt> CFGBuilder<'stmt> {
                 Stmt::With(_) => {}
 
                 // Jumps
-                Stmt::Return(_) => {}
+                Stmt::Return(_) => {
+                    let edges = Edges::always(self.cfg.terminal());
+                    self.set_current_block_stmts(&stmts[start..=end]);
+                    self.set_current_block_edges(edges);
+                    start = end + 1;
+
+                    if stmts.get(start).is_some() {
+                        let next_block = self.new_block();
+                        self.move_to(next_block);
```

Comment:
This seems a bit repetitive. We may want to abstract this somehow (e.g. store `stmts` in the current block and have a `self.branch(offset)` or similar). But I'm fine deferring this a little longer